src/adapters/__tests__/url.test.js

```diff
@@ -20,6 +20,21 @@ describe('URL', () => {
         urlAdapter.set('bar', 'baz')
         expect(getParams().get('bar')).toBe('baz')
       })
+
+      test('keeps the url hash when setting data', () => {
+        window.history.pushState({}, '', '/url/#hash')
+        urlAdapter.set('foo', 'bar')
+
+        expect(window.location.href).toEqual('http://localhost/url/#hash?foo=bar')
```

Comment:
For consistency with other tests, it would be better to only check for the hash (with a custom utility such as `getHash` on the same model than [`getParams`](https://github.com/sarahdayan/browserstore.js/blob/624e3bd58aa74e989f3636cf3aef7a773ac2b452/src/adapters/__tests__/url.test.js#L3)).